docs/modules/ROOT/partials/install/prepare-installer.adoc

```diff
@@ -1,5 +1,3 @@
-For the following steps, change into a clean directory (for example a directory in your home).
```

Comment:
This needs to stay in the partial or the Exoscale instructions need to be updated accordingly.

Reply:
🤦‍♀️ forgot to move the commodore init into a partial